pkg/controller/datavolume-controller.go

```diff
@@ -1904,7 +1905,6 @@ func (r *DatavolumeReconciler) getCloneStrategy(dataVolume *cdiv1.DataVolume) (*
 	sourcePvc, err := r.findSourcePvc(dataVolume)
 	if err != nil {
 		if k8serrors.IsNotFound(err) {
```

Comment:
Just in case, you left an empty condition here.

Reply:
right thanks :)